test_integrations/vectorhub/test_dataset_vectorize.py

```diff
@@ -1,10 +1,9 @@
-from unittest import result
 from relevanceai.datasets import mock_documents
 from relevanceai.http_client import Client, Dataset
 
 
 def test_dataset_vectorize(test_dataset: Dataset):
     results = test_dataset.vectorize(text_fields=["sample_3_description"])
```

Comment:
Can you extend this test for image fields just to be sure? 

You can use the documents here if you want example images: https://colab.research.google.com/drive/12GiCo9r0JpFqcg5Tmae28Wz0VPMpHul_?usp=sharing